packages/core/src/anchor/ethereum/remote-cas.ts

```diff
@@ -36,6 +38,7 @@ function parseResponse(streamId: StreamID, tip: CID, json: unknown): AnchorEvent
     }
   } else {
     if (parsed.status === AnchorRequestStatusName.COMPLETED) {
+      Metrics.count('cas_request_success', 1)
```

Comment:
nitpick: can we call this `cas_request_completed`?  I want to make sure to keep things clear on the distinction between an anchor request being successfully created against the CAS vs the request being fully processed and the anchor performed successfully.

Reply:
Appreciate the nitpicks :)